wiki/History_of_osu!/Online_Rankings/en.md

```diff
@@ -0,0 +1,47 @@
+---
+tags:
+  - history
+  - leaderboard
+  - rank 1
+  - rank #1
+  - rankings
+  - top player
+---
+
+# History of Online Rankings
+
+Initial versions of osu! was an mostly isolated experience, on [October 6, 2007](https://osu.ppy.sh/community/forums/topics/15?n=29), support for public online rankings was added for [osu!](/wiki/Game_mode/osu!). Since then, all [game modes](/wiki/Game_mode) has recieved [online rankings](https://osu.ppy.sh/rankings/osu/performance).
+
+Numerous players has held rank #1 in each game mode under different ranking systems. The current system used by all game modes for online rankings is [performance points (pp)](/wiki/Performance_points).
+
+## ![][osu!] osu!
+
+*Full ranking history page: [osu!](osu!)*
```

Comment:
```suggestion
*Main page: [osu!](osu!)*
```
should use main page hatnote style, othwerwise format using for see

Reply:
Resolved with https://github.com/ppy/osu-wiki/pull/4720/commits/caccb677c88639e0e96ce9a84d8436f54ad442a9